src/win32.mak

```
@@ -117,33 +102,24 @@ check-host-dc:
 debdmd: check-host-dc debdmd-make
 
 debdmd-make:
-	$(DMDMAKE) "DDEBUG=-debug -g -unittest" "DOPT=" $(TARGETEXE)
+	$(DMDMAKE) "ENABLE_DEBUG=1" "DFLAGS=-unittest" $(TARGETEXE)
 
 reldmd: check-host-dc reldmd-make
 
 reldmd-make: $(GEN)\build.exe
-	$(RUN_BUILD) "DDEBUG=" "ENABLE_RELEASE=1" $(TARGETEXE)
+	$(RUN_BUILD) "ENABLE_RELEASE=1" $(TARGETEXE)
 
+# Not ENABLE_RELEASE to avoid -inline
 profile:
-	$(DMDMAKE) "DDEBUG=" "DOPT=-O -release -profile" $(TARGETEXE)
+	$(DMDMAKE) "ENABLE_PROFILE=1" "DFLAGS=-O -release" $(TARGETEXE)
 
-trace:
-	$(DMDMAKE) "DDEBUG=-debug -g -unittest" "DOPT=" $(TARGETEXE)
+trace: debdmd-make
 
 unittest:
-	$(DMDMAKE) "DDEBUG=-debug -g -unittest -cov" "DOPT=" $(TARGETEXE)
+	$(DMDMAKE) "ENABLE_DEBUG=1" "DFLAGS=-unittest -cov" $(TARGETEXE)
```

```suggestion
	$(DMDMAKE) "ENABLE_DEBUG=1" "ENABLE_UNITTEST=1" "DFLAGS=-cov" $(TARGETEXE)
```

An `ENABLE_COVERAGE` option would be useful here.

---

Fixed. `ENABLE_COVERAGE` exists already.